assets/src/edit-story/components/canvas/page.js

```diff
@@ -40,6 +41,24 @@ const Element = styled.div`
 
 function Page() {
 	const [ targetEl, setTargetEl ] = useState( null );
+	const [ editorUnits, setEditorUnits ] = useState( { width: 0, height: 0 } );
+	const backgroundRef = useRef( null );
+
+	// Measure the editor's size and handle resize events.
```

Comment:
Does the page background resize? Isn't it fixed?

Reply:
I'm confirming that part right now. But for safety I added resizer. Was there a specific discussion w.r.t. to the editor's page size? Perhaps for the older editor? I wasn't sure if 434x775 was based on anything specific.

Reply:
But assuming we just deal with the fixed 434x775, are we ok to use `PAGE_WIDTH` and `PAGE_HEIGHT` throughout our code for this? I can delay `units` part for a bit then.

Reply:
For P1 I think this is a safe assumption. @pbakaus?

Reply:
This is actually an open question regarding the overall responsive design strategy (created in #3920). For this ticket, I think it's safe to assume that the page never changes size, and if it does, we'll handle it in #3920.

Reply:
Page canvas shouldn't change in the editor, for now.

Reply:
Great! Rolling back this code for now then.